### ISSUE TITLE
number of columns 

### ISSUE THREAD

**Issue by scooper1** [idittansikte/bootstrap-grid-card#1]
reading the docs there should be 12 columns
I can only set out 11 

if I set col-4 three times in a row it steps the last col-4 on to next row
if  set col, col, col on a row I get the full-width page
I can set col-4, col-4, col-3  and get the three items on the same row and one bank column

I would expect the  col , col, col and col-4, col-4, col-4 I would expect the col and col-4 to be the same width but the col-4 is a few pixels wider 

```
views:
  - title: cameras
    path: cameras
    badges: []
    panel: true
    cards:
      - type: 'custom:bootstrap-grid-card'
        cards:
          - type: row
            cards:
              - type: picture-entity
                class: col
                entity: binary_sensor.shelly_shem_d3b509_over_power
                show_state: false
                hold_action:
                  action: more-info
                show_name: true
                name: Back
                camera_image: camera.back
                camera_view: live
              - type: picture-entity
                class: col
                entity: sensor.nest_doorbell_chime_time
                show_name: true
                show_state: false
                name: Doorbell
                camera_image: camera.front_door
                camera_view: live
              - type: picture-entity
                class: col
                entity: sensor.nest_doorbell_chime_time
                show_name: true
                show_state: true
                camera_image: camera.doorbell_chime_image
                name: Doorbell Chime
          - type: row
            cards:
              - type: picture-entity
                class: col-4
                entity: sensor.nest_doorbell_sound_time
                show_name: true
                show_state: true
                camera_image: camera.doorbell_sound_image
                name: Doorbell Sound
              - type: picture-entity
                class: col-4
                entity: sensor.nest_doorbell_motion_time
                show_name: true
                show_state: true
                camera_image: camera.doorbell_motion_image
                name: Doorbell Motion
              - type: picture-entity
                class: col-4
                entity: sensor.nest_doorbell_person_time
                show_name: true
                show_state: true
                camera_image: camera.doorbell_person_image
                name: Doorbell Person
          - type: row
            cards:
              - type: picture-entity
                class: col-6
                entity: automation.car_presence
                show_state: false
                hold_action:
                  action: more-info
                show_name: true
                camera_image: camera.front_cam2
                camera_view: live
                name: Front
              - type: picture-entity
                class: col-5
                entity: sensor.nest_doorbell_chime_time
                show_name: true
                show_state: false
                name: Doorbell
                camera_image: camera.nest_hello
                camera_view: live

```
<img width="1913" alt="Screenshot 2021-02-10 232707" src="https://user-images.githubusercontent.com/14822945/107586294-8658e900-6bf7-11eb-8fd8-1c6a1213a115.png">

**Comment by idittansikte** [comment]
Thanks for sharing! I have that behavior too and found the problem and pushed a fix for it.

**Comment by idittansikte** [comment]
Fixed in v0.1.0.
